Modules/socketmodule.c

```diff
@@ -2797,7 +2797,9 @@ static PyObject *
 sock_send(PySocketSockObject *s, PyObject *args)
 {
     char *buf;
-    int len, n = -1, flags = 0, timeout;
+    int flags = 0, timeout;
+    ssize_t n = -1;
```

Comment:
I think use `Py_ssize_t` is more portable. It's defined [here](https://github.com/python/cpython/blob/2.7/Include/pyport.h#L182).
`Py_buffer` also use this to declare its fields.